### PR TITLE
docs tweaks + PG docs starter

### PR DESCRIPTION
**Pull request by snopoke**
I started by just wanting to simplify the layout and make it more accessible but ended up also adding some PG docs to test a 'per service' layout.

Probably easiest to review per commit.

### CODE REVIEW

File: docs/services/postgresql.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# PostgreSQL
+{:.no_toc}
+
+* TOC
+{:toc}
+
+## Usage in CommCare
+
+PostgreSQL is one of the primary databases that CommCare users.
```
Comment (orangejenny) [inline docs/services/postgresql.md]: users => uses

File: docs/index.md [hunk 2]
```diff
@@ -51,14 +59,14 @@ but in steady state as well.
 
 If you or your organization is hosting or interested in hosting
 its own CommCare server environment,
-we strongly suggested you read our [Hosting Considerations](hosting-considerations) page
+we strongly suggested you read our [Hosting Considerations](system/hosting-considerations.md) page
```
Comment (pr33thi) [inline docs/index.md]: suggested > suggest

File: docs/services/postgresql.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  - this is only required for installations that require data sharding due to the scale
+- `commcarehq_p[N]`
+  - database partition storing cases, forms, ledgers and messaging data
+  - there can be any number of these depending on the scale of the installation
+  
+For small scale installations many of these databases can be combined into a single database.
+  
+{::comment}
+Would be nice to get a diagram in here.
+{:/comment}
```
Comment (pr33thi) [inline docs/services/postgresql.md]: Was this meant to be included?

**Comment by pr33thi** [comment]
Can you run `make`, to update changelog/index.md? It looks like it got removed from the repo at some point, not sure why, but this seems like a good time to add it back in.

**Pull request merged**